Directory.Build.targets

```diff
@@ -2,4 +2,10 @@
 <Project>
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" />
   <Import Project="eng\targets\Imports.targets" />
+
+  <ItemGroup>
+    <!-- Make sure Public API files show in Solution Explorer. -->
+    <None Include="@(PublicAPI)" />
```

Comment:
Is this always required for additional files? As in what regressed here that caused the underlying issue?

Reply:
The underlying bug impacts any situation where the following occurs:

1. The user disables automatic inclusion of `<None>` items, which we do for dotnet/roslyn here:

    https://github.com/dotnet/roslyn/blob/4bc58b2494674c2338fb5e149d7631f10a5deaf7/eng/targets/Settings.props#L43

2. The code fix applies changes to an additional file which was not explicitly included as a `<None>` item

It would also occur for cases where the additional item is outside the directory tree but not linked to the project as a `<None>` item. It may suffice to include the item as a _visible_ item, as opposed to specifically requiring `None`.

Reply:
Have you tried removing `<EnableDefaultNoneItems>false</EnableDefaultNoneItems>`. It's not clear why we would need to change the .NET SDK default.